numpy/core/src/umath/ufunc_object.c

```diff
@@ -5888,6 +5888,13 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
 
     NPY_BEGIN_THREADS_DEF;
 
+    if (ufunc->core_enabled) {
+        PyErr_Format(PyExc_TypeError,
+            "numpy.ufunc.at does not support ufunc with non-trivial "\
```

Comment:
Just a nit, but the `\` is not necessary :).

We could probably make the error message friendlier by printing out the ufunc name rather than "numpy.ufunc.at".  The name is far more interesting than the actual core signature!